cl/audio/tests.py

```diff
@@ -1,12 +1,43 @@
+from datetime import datetime
+from unittest import mock
+
 from django.urls import reverse
 from lxml import etree
 
+from cl.audio.factories import AudioWithParentsFactory
 from cl.audio.models import Audio
 from cl.lib.test_helpers import IndexedSolrTestCase, SitemapTest
 from cl.search.models import SEARCH_TYPES
+from cl.tests.fixtures import ONE_SECOND_MP3_BYTES, SMALL_WAV_BYTES
 
 
 class PodcastTest(IndexedSolrTestCase):
+
+    fixtures = [
+        "test_court.json",
+        "judge_judy.json",
+        "test_objects_search.json",
+        "authtest_data.json",
+    ]
```

Comment:
I'm confused....why do you need fixtures here that weren't here before?

Reply:
Yeah, its too override the `IndexedSolrTestCase` fixtures which include `test_objects_audio.json` 

This way I could keep the other fixtures I hadn't yet removed.